packages/react-storage/src/components/StorageBrowser/controls/hooks/useStatusDisplay.ts

```diff
@@ -0,0 +1,37 @@
+import { StatusDisplayProps } from '../../composables/StatusDisplay';
+import { useControlsContext } from '../../controls/context';
+import { displayText } from '../../displayText/en';
+
+export interface UseStatusDisplay {
```

Comment:
Nit: This interface name is a bit confusing since the hook itself return the same-named interface.

Reply:
I sorta copied existing pattern from paginate hook. I can update